packages/blockchain-wallet-v4-frontend/src/data/components/simpleBuy/sagas.ts

```diff
@@ -499,13 +521,50 @@ export default ({
     }
   }
 
+  const fetchSBCardSDD = function * (
+    billingAddress: SBBillingAddressFormValuesType
+  ) {
+    let card: SBCardType
+    try {
+      yield put(A.fetchSBCardLoading())
+      const order = S.getSBLatestPendingOrder(yield select())
+      if (!order) throw new Error(NO_ORDER_EXISTS)
+      const currency = getFiatFromPair(order.pair)
+      if (!currency) throw new Error(NO_FIAT_CURRENCY)
+
+      const userDataR = selectors.modules.profile.getUserData(yield select())
```

Comment:
Same as previous comment. `yield select(selector)` should work, right?

Reply:
yeah we use somewhere this way somewhere other way, no consensus about which one is desired, but yeah it will work in both cases :)